TRYING_TO_LOAD_RANDOM_DATASET Signal_4000_D_0_constant_24_None_0.1_0 data/ARTIFICIAL_DATA/4000/Signal_4000_D_0_constant_24_None_0.1_0
LAOD_FAILED_TRYING_TO_GENERATE_RANDOM_DATASET Signal_4000_D_0_constant_24_None_0.1_0
GENERATING_RANDOM_DATASET Signal_4000_D_0_constant_24_None_0.1_0
TREND 0.0976270078546495 0.43037873274483895 0.20552675214328775
Traceback (most recent call last):
  File "/home/circleci/project/tests/bugs/issue_70/test_artificial_filter_seasonals_day.py", line 8, in <module>
    b1 = tsds.generate_random_TS(N = 4000 , FREQ = 'D', seed = 0, trendtype = "constant", cycle_length = 24, transform = "None", sigma = 0.1, exog_count = 0);
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/project/pyaf/Bench/TS_datasets.py", line 333, in generate_random_TS
    tsspec = generate_random_TS_real(N , FREQ, seed, trendtype, cycle_length, transform, sigma, exog_count, ar_order)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/project/pyaf/Bench/TS_datasets.py", line 391, in generate_random_TS_real
    df_train['Signal'] = apply_transform(pos_signal , transform)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/circleci/project/pyaf/Bench/TS_datasets.py", line 297, in apply_transform
    transformed = transformed.astype(np.float64)
                  ^^^^^^^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'astype'
